### PR TITLE
A couple of improvements to the issue templates

### PR DESCRIPTION
**Pull request by quyykk**
This PR does the following:

- Adds a new template for wiki reports
- ~~Enables users to ignore the template if necessary. I find this better than forcing a template that might not fit and the requiring the user to edit the post after the fact.~~
- Rename yaml -> yml.

### CODE REVIEW

**Review by tibetiroka — APPROVED** [review]
Grammar looks OK.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1 +0,0 @@
-blank_issues_enabled: false
```
Comment (tehhowch) [inline .github/ISSUE_TEMPLATE/config.yml:1]: This was added because users would ignore the template, rather than pick the one that they should have used

Comment (quyykk) [reply .github/ISSUE_TEMPLATE/config.yml:1]: Yes I know but sometimes the template doesn't fit and in that case it's annoying. It's also a rather small link at the bottom under the templates.

Additionally, even with the templates we see people ignoring them by just putting random stuff in the boxes and then editing them out.

File: .github/ISSUE_TEMPLATE/wiki-issue.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  - type: markdown
+    attributes:
+      value: Thank you for filing a wiki report!
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the issue
+      description: Provide a clear and concise description of what the issue is.
+    validations:
+      required: true
```
Comment (petervdmeer) [inline .github/ISSUE_TEMPLATE/wiki-issue.yml:15]: Can we also add a field with something like `Link to the exact online content location`, similar to how bugreports request the exact game-version?

File: .github/ISSUE_TEMPLATE/wiki-issue.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for filing a wiki report!
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the issue
+      description: Provide a clear and concise description of what the issue is.
```
Comment (petervdmeer) [inline .github/ISSUE_TEMPLATE/wiki-issue.yml:13]: The bug-report template also asks for a "Expected Behavior". If we want to follow that same style for wiki/documentation bug-reports, then we could add a field here for `Expected content`.

File: .github/ISSUE_TEMPLATE/wiki-issue.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+name: Wiki Issue
```
Comment (Amazinite) [inline .github/ISSUE_TEMPLATE/wiki-issue.yml:1]:
```suggestion
name: Documentation Issue
```

File: .github/ISSUE_TEMPLATE/wiki-issue.yml [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+description: Something is wrong with the wiki.
```
Comment (Amazinite) [inline .github/ISSUE_TEMPLATE/wiki-issue.yml:2]:
```suggestion
description: There is an error in documentation, such as documentation being wrong or outdated. Documentation may include the wiki on GitHub, RFCs, style guides, or other such pages/documents.
```

File: .github/ISSUE_TEMPLATE/wiki-issue.yml (hunk 2 above)
Comment (Amazinite) [reply .github/ISSUE_TEMPLATE/wiki-issue.yml:15]: With the suggested change to describe more documentation sources than just the wiki, having a field for specifying where the documentation is, as @petervdmeer says, would be helpful.

**Pull request merged**